src/gui/addnewtorrentdialog.h

```diff
@@ -38,6 +38,7 @@
 
 #include "base/bittorrent/infohash.h"
 #include "base/bittorrent/torrentinfo.h"
+#include "base/bittorrent/session.h"
```

Comment:
@glassez Well, we do need it, or something like it. Not every option in `BitTorrent::AddTorrentParams` is represented by UI in `AddNewTorrentDialog`. In particular, `firstLastPiecePriority` and `sequential`, both of which can be specified from the command line, do not have options in the dialog. But we still want them to be enabled if the user adds the torrent from the command line and specifies them using `--first-and-last` or `--sequential`, regardless of whether the dialog is invoked or not.

I opted to have `BitTorrent::AddTorrentParams` be a member variable because that was the safest way to ensure every option it specifies gets used when the torrent is actually added when `BitTorrent::Session::instance()->addTorrent()` is called by `accept()`. This is particularly relevant in case someone adds more fields to `BitTorrent::AddTorrentParams`, because those will be respected too -- it gives us a degree of future proofing and is less error prone.

That said, it is true that currently the only options we care about passing through the dialog *right now* that don't have a control in its UI are `firstLastPiecePriority` and `sequential`. In order to eliminate `m_torrentParams` we could add boolean member variables to `AddNewTorrentDialog` that just store those values, and then construct a new `BitTorrent::AddTorrentParams` in the `accept()` method.

However if the only reason to eliminate `m_torrentParams` is to remove an `#include` from the header file, I don't think it's a net win given that keeping it around has the benefits I listed above. But I'm not wedded to either approach. If you want me to eliminate `m_torrentParams` I'll do it.